entry_types/paged/packages/pageflow-paged/src/frontend/history/PushStateAdapter.js

```diff
@@ -0,0 +1,39 @@
+import $ from 'jquery';
```

Comment:
I am so sorry for this , I have already struggled  too much with this. On my local it looks perfectly fine  no duplicate files but here somehow its like this.
I had this weird behaviour triggered because of filenames ignoreCase stuff.